controllers/config/constants.go

```diff
@@ -2,17 +2,36 @@ package config
 
 // Kubernetes built-in well-known constants
 const (
-	// LabelAppName is the name of the component application
+	// LabelAppName is the name of the component application.
 	LabelAppName = "app.kubernetes.io/name"
-	// LabelAppInstance identifies resources related to a unique Cluster deployment
+	// LabelAppInstance identifies resources related to a unique Cluster deployment.
 	LabelAppInstance = "app.kubernetes.io/instance"
+	// LabelAppManagedBy is the controller/user who created the resource.
+	LabelAppManagedBy = "app.kubernetes.io/managed-by"
 )
 
+
+// CSI resource labels
 const (
-	// LabelName is the name of the product
-	LabelName = "product"
-	// Product base name
+	// LabelName is the name of the product.
+	LabelProduct = "product"
+	// LabelRelease is the current version of the application.
+	LabelRelease = "release"
+)
+
+// CSI resource label values
+// Note: Release should be updated here to reflect the current CSI version across all the resources.
+const (	
+	// Product is the name of the application.
 	Product = "ibm-spectrum-scale-csi"
+	// Release is the current version of the application.
+	Release  = "2.3.0"
```

Comment:
Since we are using 2.4.0 for resizer, please change to 2.4.0

Reply:
Changed the value to `ibm-spectrum-scale-csi-operator` as in Ansible templates.